include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
I don't understand here. If you define pthread wrappers and pthread_xxxx_t types in "windows/osd.h", no changes are needed outside the "windows" directory.

In addition, how changing the parameter types in the prototype to "struct pthread_xxx_t" makes life any easier, not considering the confusion it may bring?